api.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type RecurseAPI struct {
+	rcAPIURL string
+}
+
+type RecurserProfile struct {
+	Stints []Stint
+}
+
+type Stint struct {
+	In_progress bool
+}
+
+func (ra *RecurseAPI) userIsCurrentlyAtRC(accessToken string, email string) bool {
+	currentlyAtRC := false
+
+	resp, err := http.Get(ra.rcAPIURL + "/profiles/" + email + "?access_token=" + accessToken)
+
+	if err != nil {
+		log.Printf("Got the following error while checking if the user was active at RC: %s\n", err)
+		return currentlyAtRC
+	}
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	//Parse the json response from the API
+	profile := RecurserProfile{}
+	json.Unmarshal([]byte(body), &profile)
+
+	log.Println("Recurser Profile: ", profile)
+
+	currentlyAtRC = profile.Stints[0].In_progress
```

Comment:
Even though the JSON response uses a lower case "in_progress", Go requires us to use the upper case "In_progress" inside the `type Stint struct`. I guess there's some Go magic going on that converts the casing 🤷

Reply:
Each RC profile has a list of their stints listed from newest to oldest.